test/e2e/sync_plugin_test.go

```diff
@@ -135,13 +135,15 @@ func setupThroughJenkinsLaunch(t *testing.T, ta *testArgs) *testArgs {
 	setupClients(ta.t)
 
 	randomTestNamespaceName := generateName(testNamespace)
-	ta.ns = randomTestNamespaceName
-	_, err := projectClient.ProjectV1().ProjectRequests().Create(context.Background(), &projectv1.ProjectRequest{
-		ObjectMeta: metav1.ObjectMeta{Name: randomTestNamespaceName},
-	}, metav1.CreateOptions{})
+	if len(ta.ns) == 0 {
```

Comment:
did you run `gofmt` @alicerum ?

if so, I've been running into the same  issue in this repo.  For some reason, `gofmt` works differently for golang files in this repo.

@adambkaplan @akram FYI

Reply:
aside from gofmt, move `randomTestNamespaceName := generateName(testNamespace)` inside the `if len(ta.ns) == 0` block, and switch to `ta.ns = generateName(testNamespaace)`

Reply:
`gofmt` does nothing for me here :/

Reply:
I think we should address the formatting issue in a separate PR - seems to be a common issue (with perhaps the same root cause).

Reply:
Damn ... unfortunately:

```
gmontero ~/go/src/github.com/openshift/jenkins-sync-plugin  (add-sample-side-car-e2es)$ git rm .editorconfig 
rm '.editorconfig'
gmontero ~/go/src/github.com/openshift/jenkins-sync-plugin  (add-sample-side-car-e2es)$ gofmt -w ./test/
gmontero ~/go/src/github.com/openshift/jenkins-sync-plugin  (add-sample-side-car-e2es)$ git status 
On branch add-sample-side-car-e2es
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	deleted:    .editorconfig

gmontero ~/go/src/github.com/openshift/jenkins-sync-plugin  (add-sample-side-car-e2es)$ 
```

Reply:
though maybe it renders in github with the right spacing? ... I noticed in my goland I have 4 space indentation now, even though gofmt does not "change" the file

Reply:
but so it does not get lost in the shuffle @alicerum still make the change I noted at https://github.com/openshift/jenkins-sync-plugin/pull/408#discussion_r766768340 

orthogonal to gofmt

Reply:
@gabemontero already pushed the changed

as for editor, it only shows github how to show tabs, @adambkaplan and me checked that actual go files contain tab identation and everything is alright :D

Reply:
ok thanks for confirming my "though maybe it renders..." speculation

I double confirmed when pushing a commit to #406 which removes .editorconfig ... the tabbing looks better in that PR